README.md

```diff
@@ -96,8 +97,82 @@ if `glslc` or `xxd` are not present, a precompiled version of the shaders is sto
 <a href="https://aur.archlinux.org/packages/vkvg"><img src="https://img.shields.io/aur/version/vkvg"></a>
 <a href="https://mpr.hunterwittenborn.com/packages/vkvg"><img src="https://img.shields.io/badge/mpr-v0.3.0--beta-blue"></a>
 
+## Meson Build
```

Comment:
one more level from 'building' (an additional '#'), also I would have preferred the meson build to appear after the original cmake. (at least for now).